zerver/lib/digest.py

```diff
@@ -371,7 +371,7 @@ def bulk_handle_digest_email(user_ids: List[int], cutoff: float) -> None:
                 "zerver/emails/digest",
                 user.realm,
                 to_user_ids=[user.id],
-                from_name="Zulip Digest",
+                from_name=f"{settings.SERVICE_NAME} Digest",
```

Comment:
@alya I suppose there's a question of whether we want to tag this for translation like we do the others below.